app/src/test/java/org/simple/clinic/teleconsultlog/shareprescription/TeleconsultSharePrescriptionUiRendererTest.kt

```diff
@@ -16,7 +16,7 @@ class TeleconsultSharePrescriptionUiRendererTest {
   private val prescriptionDate = LocalDate.parse("2020-10-01")
 
   val model = TeleconsultSharePrescriptionModel
-      .create(patientUuid = patientUuid, prescriptionDate = prescriptionDate)
+      .create(patientUuid = patientUuid, prescriptionDate = prescriptionDate, downloadButtonState = ButtonState.NOT_DOWNLOADING)
```

Comment:
There doesn't seem to be any UI renderer tests for checking download button state changes